### PR TITLE
Change to replace secure data values with blank strings

### PR DESCRIPTION
**Pull request by gjones-ch**
Change to replace secure data values with blank strings when writing data to the payment reconciliation collection

### CODE REVIEW

File: assets/product_code.yml [hunk 1]
```diff
@@ -1,2 +1,3 @@
 product_code:
-  cic-report: 16706
\ No newline at end of file
+  cic-report: 16706
+  ProApp1: 16800
```
Comment (divya051988) [inline assets/product_code.yml]: Can we please keep the product_code format consistent ?

File: service/service_test.go [hunk 2]
```diff
@@ -144,6 +175,8 @@ func TestStart(t *testing.T) {
 
 								var ptr models.PaymentTransactionsResourceDao
 								mockTransformer.EXPECT().GetTransactionResource(pr, pdr, paymentResourceID).Return(ptr, nil).Times(1)
+								//So(ptr.Email, ShouldEqual, "")
```
Comment (divya051988) [inline service/service_test.go]: Please remove commented out code.

**Comment by divya051988** [comment]
Have a read at https://github.com/companieshouse/styleguides/pull/43 Make sure the test methods are either called TestUnit or TestIntegration .

File: service/service_test.go [hunk 3]
```diff
@@ -219,6 +250,82 @@ func TestStart(t *testing.T) {
...
+
+	productMap, err := createProductMap()
+	if err != nil {
+		log.Error(fmt.Errorf("error initialising productMap: %s", err), nil)
+	}
+
+	svc := createMockService(productMap, mockPayment, mockTransformer, mockDao)
+
+	created := data.Created{
+		Email: "demo@ch.gov.uk",
```
Comment (divya051988) [inline service/service_test.go]: Could you please change this to a test email which does not exist.

File: service/service.go [hunk 4]
```diff
@@ -284,6 +289,24 @@ func (svc *Service) Start(wg *sync.WaitGroup, c chan os.Signal) {
 	log.Info("Service successfully shutdown", log.Data{"topic": svc.Topic})
 }
 
+// We need a function to mask potentially sensitive data fields in the event it' a secure application.
```
Comment (dalebradley) [inline service/service.go]: it's

**Pull request merged**